### DISCUSSION TITLE
有源码相关的注释文档吗？ 或者项目架构的文档？

### DISCUSSION THREAD

**Discussion by lmmqxyx404** [shadowsocks/shadowsocks-rust#1007] [Q&A]
我想深入学习一下 shadowsocks 协议的开发流程

**Comment by zonyitoo** [comment]
https://shadowsocks.org/guide/what-is-shadowsocks.html

**Reply by lmmqxyx404** [reply]
shadowsocks 原理我比较清楚，但是这个ss-rust项目比较复杂。我在win10上都无法编译，甚至都不知道编译产物。
您刚刚发的链接我也看了，但说实话，对我的帮助并不大。请问，有更细致的项目介绍吗？

**Reply by zonyitoo** [reply]
I believe it should be able to compile on Win10, because the CI was always passed.

> 请问，有更细致的项目介绍吗？

I don't think so.

**Reply by lmmqxyx404** [reply]
Excuse me, I'm not trying to pick a fight. Before I start the ps1 script, I just run ```cargo clean```.But I still got the error.
![屏幕截图 2022-12-06 164422](https://user-images.githubusercontent.com/44399714/205863737-b72c4a4e-1ab3-461a-8cfa-eaa8e3bb73f0.png)
Feel free to point out possible stupid errors that I could make.
Is there a real-time communication tool associated with the project, discord or telegram?

**Reply by lmmqxyx404** [reply]
I can guarantee that  the project source code is the latest .

**Reply by zonyitoo** [reply]
Upgrade your `rustc`'s version to the latest.

**Reply by lmmqxyx404** [reply]
Thanks a lot.Your advice works well.
